### PR TITLE
SUP-179 | @rebeccahongsf | Theme and style masthead, navigation, and logo

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Theme and style masthead, navigation, and logo

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Spin up local
3. Review masthead and compare to figma mocks
4. Review code

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

# Associated Issues and/or People
- SUP-179

### CODE REVIEW

File: src/components/images/header-logo-lg.tsx [hunk 1]
```diff
@@ -10,6 +12,11 @@ type Props = HtmlHTMLAttributes<SVGSVGElement> & {
 
 const HeaderLogoLg = ({height = 89, width = 579, ...props}: Props) => {
   const hasPageBanner = usePageHasTopBanner()
+  const { width: windowWidth } = useWindowSize()
+  
+  if (!height || height === 89) {
+    height = windowWidth >= 1500 ? 126 : windowWidth <= 1500 ? 104 : 72;
```
Comment (pookmish) [inline src/components/images/header-logo-lg.tsx]: Is it possible to remove the height/width attributes and use classes?

**Review by pookmish — COMMENTED** [review]
Just a quick question for you to review.

File: src/components/images/header-logo-lg.tsx (hunk 1 above)
Comment (rebeccahongsf) [reply src/components/images/header-logo-lg.tsx]: @pookmish we can definitely use classes instead and set the width/height attributes to 100%! I'll tweak the conditionals and switch us over to using TW classes 🫡

File: src/components/images/header-logo-lg.tsx [hunk 2]
```diff
@@ -2,32 +2,29 @@
...
-    width *= height / 89
-  }
-
-  if (height == 89 && width != 579) {
-    height *= width / 579
-  }
+  const {width: windowWidth} = useWindowSize()
+  const height = windowWidth >= 1500 ? 126 : windowWidth <= 1500 ? 104 : 72
+  const width = (height / 89) * 579
+  const size = `w-[${width}px] h-[${height}px]`
```
Comment (pookmish) [inline src/components/images/header-logo-lg.tsx]: Dynamic classes don't work in tailwind
https://tailwindcss.com/docs/content-configuration#dynamic-class-names
Is there any way you can just do more simple classes without using the `useWindowSize`? Just some old fashion `md:w-[]` type classes?

**Review by pookmish — CHANGES_REQUESTED** [review]

File: src/components/images/header-logo-lg.tsx (hunk 2 above)
Comment (rebeccahongsf) [reply src/components/images/header-logo-lg.tsx]: Most definitely! I'll update this shortly 😃

**Review by pookmish — APPROVED** [review]

**Pull request merged**